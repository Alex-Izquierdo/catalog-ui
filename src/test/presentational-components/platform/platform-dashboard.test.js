import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import PlatformDashboard from '../../../presentational-components/platform/platform-dashboard';

describe('<PlatformDashboard />', () => {
  it('should render correctly', () => {
    expect(shallowToJson(shallow(<PlatformDashboard />))).toMatchSnapshot();
  });
});
