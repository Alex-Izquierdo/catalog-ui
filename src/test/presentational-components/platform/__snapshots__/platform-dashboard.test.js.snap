// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PlatformDashboard /> should render correctly 1`] = `
<Connect(Main)>
  <div
    style={
      Object {
        "textAlign": "center",
      }
    }
  >
    <Title
      size="md"
    >
      Search Or Select a Platform
    </Title>
  </div>
</Connect(Main)>
`;
